queries.sql

```diff
@@ -1,17 +1,99 @@
-/*Queries that provide answers to the questions from all projects.*/
+/*Queries that provide answers to the questions from  projects.*/
 
 SELECT * FROM animals WHERE name LIKE '%mon';
 
-SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31'
 
-SELECT * FROM animals WHERE neutered = true AND escape_attempts < 3;
+SELECT * FROM animals WHERE neutered =  true and escape_attempts < 3;
 
 SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
 
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 
-SELECT * FROM animals WHERE neutered =  true;S
+SELECT * FROM animals WHERE neutered =  true;
 
 SELECT * FROM animals WHERE name <> 'Gabumon';
 
-SELECT * FROM animals WHERE weight_kg BETWEEN  10.4 and 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN  10.4 and 17.3;
+
+-- transaction start
+
+ BEGIN;
+ 
+ UPDATE animals
+ SET species = 'unspecified';
+ 
+ ROLLBACK;
+ 
+ COMMIT;
+
+--  transaction end
+
+-- transaction start
+BEGIN;
+ 
+UPDATE animals
+ SET species = 'digimon'
+ WHERE name LIKE '%mon';
+ 
+UPDATE animals
+ SET species = 'pokemon'
+ WHERE species IS NULL;
+ 
+COMMIT;
+
+--  transaction end
+
```

Comment:
- Again here kindly add `SELECT` statements before and after the `COMMIT` statement to ensure the changes were made.

e.g
```
   BEGIN;
   UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon'; 
   UPDATE animals SET species = 'pokemon' WHERE species IS NULL; 

   SELECT species from animals; -- verify that change was made
   COMMIT;
   SELECT species from animals; -- verify that change persists after commit
```